client/src/components/Card.js

```diff
@@ -23,13 +48,18 @@ const Card = (props) => {
             transform: interpolate([rot, scale], trans)
           }}
         >
+<<<<<<< HEAD
+         {mode==='facts' && <Facts data={data}/>}
+         {mode==='quiz' && <Quiz data={data}/>}
+=======
           <div className="card" style={{textAlign:'left'}}>
             <h1 style={{color:'red', fontStyle:'cursive'}}>{title}</h1>
             <h3 style={{fontSize:'1rem'}}>{body}</h3>
               <footer style={{textAlign:'left'}}>{footer}
                 {link && <Button size="small" href={link} >Learn More</Button>}
               </footer>
           </div>
+>>>>>>> master
```

Comment:
These `<<<` `===` and `>>>` are from the merge and need to be removed for the code to compile

Reply:
ahh i just saw that after committing. Thanks!